src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -860,6 +860,12 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\ConstrainedExecution\PrePrepareMethodAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\ConstrainedExecution\ReliabilityContractAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\..\coreclr\nativeaot\Runtime.Base\src\System\Runtime\ExceptionHandling.cs" Condition="'$(FeatureNativeAot)' != 'true' and '$(FeatureMono)' != 'true' and ('$(Platform)' != 'x86' or '$(TargetsWindows)' != 'true')"/>
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\ExceptionServices\AsmOffsets.cs" Condition="'$(FeatureNativeAot)' != 'true' and '$(FeatureMono)' != 'true' and ('$(Platform)' != 'x86' or '$(TargetsWindows)' != 'true')"/>
```

Comment:
These are CoreCLR specific files. It would make more sense to have them included via https://github.com/dotnet/runtime/blob/main/src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj . it is going to make the condition simpler.

Reply:
Makes sense, I wasn't sure about where they really belong.

Reply:
Done